### PR TITLE
Some suggestions

### PR DESCRIPTION
**Pull request by zsteinmetz**
First of all, thanks for sharing your work in this "early" stage.

I corrected some small typos and tried to make the style more homogeneous, e.g. by continuously using the \code notation for datasets. Furthermore, I suggest to reduce the unnecessary space following 'e.g.' and 'i.e.'
One general remark regarding the discussion: To me, the comparisons with other packages could be a little bit more detailed in order to better highlight the advantages of webchem over other solutions (e.g. also in terms of usability)
The common abbreviation for the log water partitioning coefficient is K\textsubscript{OW} or P\textsubscript{OW}. You may want to use that ..

### CODE REVIEW

**Comment by eduardszoecs** [comment]
Thanks for your valuable comments.
The manuscript is currently under review and I am very busy at the moment.
I will definitively consider your comments (and acknowledge them), when revising the ms after getting the reviewer comments.

See also some directly inline comments to the PR.

File: article.tex [hunk 1]
```diff
@@ -75,34 +75,34 @@
 
 
 \section[Introduction]{Introduction}
-Before each statistical analysis, data cleaning is often required to ensure a good data quality.
-Data cleaning is the process of detecting errors and inconsistencies in data sets \citep{Chapman_2005}.
+Before each statistical analysis, data cleaning is often required to ensure good data quality.
+Data cleaning is the process of detecting errors and inconsistencies in datasets \citep{Chapman_2005}.
```
Comment (eduardszoecs) [inline article.tex:79]: Not sure about this.
http://english.stackexchange.com/questions/2120/which-is-correct-dataset-or-data-set
http://forum.wordreference.com/threads/dataset-vs-data-set.1118035/

`data set` is the more general term, while `dataset` is a more specific one (though the differences are neglectable).

File: article.tex [hunk 2]
```diff
@@ -75,34 +75,34 @@
...
+When dealing with chemical datasets (e.g.\ environmental monitoring data, toxicological data), a first step is often to validate the names of chemicals or to link them to unique codes that simplify subsequent querying and appending of compound-related physico-chemical or toxicological information.
 Several web sources provide chemical names or link them to unique codes (see also section \emph{Data sources} below).
 However, manual searching for each compound, often through a graphical web interface, is tedious, error-prone and not reproducible \citep{Peng_2009}.
 
-To simplify, robustify and automate this task, i.e. to search and retrieve chemical information from the web, we created the \pkg{webchem} package for the free and open source \proglang{R} language \citep{r_2015, Wehrens_2011}.
+To simplify, robustify and automate this task, i.e.\ to search and retrieve chemical information from the web, we created the \pkg{webchem} package for the free and open source \proglang{R} language \citep{r_2015, Wehrens_2011}.
```
Comment (eduardszoecs) [inline article.tex:86]: I did not compile your PR. But what is the backslash for?

**Comment by eduardszoecs** [comment]
Note to myself:

> Be more explicit/verbose in the README how to compile the article (rnw -> tex -> pdf)

File: article.tex (hunk 1 above)
Comment (zsteinmetz) [inline article.tex:79]: Interesting, didn't know that. Thanks for highlighting the difference. My intent was just to make it uniform over the whole document.

File: article.tex (hunk 2 above)
Comment (zsteinmetz) [inline article.tex:86]: By default, LaTeX adds extra space after each fullstop (intended for sentence periods). However, this is not wanted when using abbreviations. Therefore, you may either escape the subsequent space or toggle off the extra spacing in general using \frenchspacing in the preamble (Source: https://en.wikibooks.org/wiki/LaTeX/Tips_and_Tricks#id_est_and_exempli_gratia_.28i.e._and_e.g..29)
But maybe you can leave this to the JSS typesetters.

Comment (eduardszoecs) [inline article.tex:86]: Thanks for clarification. Didn't know that.

**Comment by eduardszoecs** [comment]
I incorporated your suggestion into the next version of the paper. Thanks for your comments (which I acknowledged).

**Pull request closed without merging**